bangazonapi/views/report.py

```diff
@@ -27,3 +27,12 @@ def report(request):
         )
 
     return render(request, "index.html", {"report_data": report_data})
+
+
+def expensive_products_report(request):
+    expensive_products = Product.objects.filter(price__gte=10000)
```

Comment:
I think you added an extra 0! It should be 1,000, not 10,000. After making the change on my local version of this branch, the report worked beautifully.